### PR TITLE
FLUT-900545 - Publish note that the SfPdfViewer.file will not work on Web platform., FLUT-896840: Update the limitations of the RTL scroll direction in the Single Page View in the UG documentation

### PR DESCRIPTION
**Pull request by immankumarsync**
## Description ##

* Published note that the SfPdfViewer.file will not work on Web platform.
* Updated the limitations of the RTL scroll direction in the Single Page View in the UG documentation

### CODE REVIEW

File: Flutter/pdf-viewer/right-to-left.md [hunk 1]
```diff
@@ -120,4 +120,6 @@ Then, import the `flutter_localizations` library, specify [localizationsDelegate
   }
 
 {% endhighlight %}
-{% endtabs %}
\ No newline at end of file
+{% endtabs %}
+
```
Comment (Deepak1799) [inline Flutter/pdf-viewer/right-to-left.md:124]: image

Please use API linking or use lower case if using in words

File: Flutter/pdf-viewer/right-to-left.md [hunk 2]
```diff
@@ -120,4 +120,6 @@ Then, import the `flutter_localizations` library, specify [localizationsDelegate
...
+N> RTL scrolling is not supported in Single Page Layout mode when using horizontal scroll direction.
```
Comment (Deepak1799) [inline Flutter/pdf-viewer/right-to-left.md:125]: Can this be changed to?

RTL scrolling is not supported in single page layout mode when the scroll direction is set to horizontal.

File: Flutter/pdf-viewer/right-to-left.md [hunk 3]
```diff
@@ -120,4 +120,6 @@ Then, import the `flutter_localizations` library, specify [localizationsDelegate
   }
 
 {% endhighlight %}
-{% endtabs %}
\ No newline at end of file
+{% endtabs %}
```
Comment (Deepak1799) [inline Flutter/pdf-viewer/right-to-left.md:123]: Please API link for this too. General words may cause confusion 
image

**Review by Deepak1799 — CHANGES_REQUESTED** [review]

File: Flutter/pdf-viewer/right-to-left.md (hunk 3 above)
Comment (immankumarsync) [reply Flutter/pdf-viewer/right-to-left.md:123]: Added API to avoid confusion.

File: Flutter/pdf-viewer/right-to-left.md (hunk 1 above)
Comment (immankumarsync) [reply Flutter/pdf-viewer/right-to-left.md:124]: Added API links

**Review by Deepak1799 — CHANGES_REQUESTED** [review]
API linking is not proper

image

**Comment by immankumarsync** [comment]
> API linking is not proper
> 
> image

image
The horizontal value has no separate link.

Even when I click the link in the right it navigates to the below error page.
image

**Review by Deepak1799 — APPROVED** [review]

**Pull request merged**